.gitignore

```diff
@@ -9,6 +9,7 @@ bower_components/
 coverage/
 dist/
 kubernetes/
+tools/
```

Comment:
Since the godep tool is installed via "npm install", I would think it should remain even after gulp clean is executed. Do you think it would help if we make the directory hidden (e.g. .tools)?